packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -302,3 +303,53 @@ export const isTestProfileStableFlavor = (): boolean => {
     throw 'Faled to find Communication React Flavor env variable';
   }
 };
+
+export interface StubOptions {
+  // Stub out all timestamps in the chat message thread.
+  stubMessageTimestamps?: boolean;
+  // Disable tooltips on all buttons in the call control bar.
+  dismissTooltips?: boolean;
+}
+
+/**
+ * A helper to take stable screenshots.
+ *
+ * USE THIS INSTEAD OF page.screenshot()
+ */
+export async function stableScreenshot(
```

Comment:
The whole comment and naming of this function is misleading.
At a glance, it looks like this should be the defacto method for taking screenshots. However, what if my test requires the tooltip to be visible?

Reply:
You still use this method.

Tooltips are only dismissed if you call it as `stableScreenshot(page, {disableTooltips: true})`.
Tooltips are not dismissed if you call it as `stableScreenshot(page, {})`.

The only thing it definitely does every time is wait for fonts to load (we might wait for more such universal things in the future).

Reply:
didn't notice that optional param. (Y)